server/src/cg/group4/server/database/query/RequestPlayerData.java

```diff
@@ -25,35 +24,45 @@
      * @param id The id of the player to be retrieved.
      */
     public RequestPlayerData(final String id) {
-        cId = id;
```

Comment:
Did that to seperate the insertUser to a new method, but works better in general ;)